@navikt/core/react/src/layout/page/Page.tsx

```diff
@@ -11,7 +11,9 @@ export interface PageProps extends React.HTMLAttributes<HTMLElement> {
    */
   as?: "div" | "body";
   /**
-   * Background color. Accepts a color token.
+   * Background color.
+   * Accepts a [background color token](https://aksel.nav.no/grunnleggende/styling/design-tokens#753d1cf4d1d6)
+   * as well as the surface color token "surface-subtle".
```

Comment:
```suggestion
   * Accepts a [background color token](https://aksel.nav.no/grunnleggende/styling/design-tokens#753d1cf4d1d6).
```
https://github.com/navikt/aksel/pull/2987